pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -61,7 +61,7 @@ if [ -z "${ctrd_cgroup_cpus_limit}" ]; then
 fi
 
 CGROUPS="cpuset cpu cpuacct blkio memory devices freezer net_cls perf_event net_prio hugetlb pids systemd "
-EVESRVICES="sshd wwan wlan lisp guacd pillar vtpm watchdog xen-tools "
+EVESRVICES="sshd edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools "
```

Comment:
This should also NOT be needed if we make this edgeview container loadable on demand.